src/you_get/extractors/mgtv.py

```diff
@@ -75,8 +75,9 @@ def prepare(self, **kwargs):
                 size_this_stream = 0
                 stream_fileid_list = []
                 for i in segment_list_this:
-                    _, container_this_stream, size_this_seg = url_info(i)
-                    size_this_stream += size_this_seg
+                    if 'json_output' not in kwargs and not kwargs['json_output']:
```

Comment:
I talked to the author: He DOES NOT want to get the information of every segment to reduce time. This is not what WE want here...